src/pages/TeamView/TeamView.js

```diff
@@ -0,0 +1,133 @@
+import React, { useState } from 'react';
+import { useQuery, gql } from '@apollo/client';
+import { useSelector } from 'react-redux';
+import { Box, Button, Heading, Input, FormControl } from '@chakra-ui/react';
+
+const GET_TEAM_INFO = gql`
+  query getTeam($id: String!) {
+    getTeam(id: $id) {
+      name
+      organization
+      id
+      amountRaised
+      itemsSold
+    }
+  }
+`;
+
+const TeamView = () => {
+  const {
+    user: { teamId },
+  } = useSelector(state => state.auth);
+
+  return teamId ? (
+    <TeamOverview teamId={teamId} />
+  ) : (
+    <div>
+      <h1>Welcome</h1>
+      {'User is not part of a team (todo: implement this view)'}
+    </div>
+  );
+};
+
+const InviteTeamMembers = () => {
+  const [inputList, setInputList] = useState([{ email: '' }]);
```

Comment:
I think we can just use a plain string array here rather than an array of objects